### ISSUE TITLE
When going to rocket chat home page, blank screen comes with loading icon stuck and frozen.

### ISSUE THREAD

**Issue by FaizanZahid** [RocketChat/Rocket.Chat#12086]
When i do same action on incognito, it works, but here are the JS errors i am getting when i m normally trying to open our homepage.

![image](https://user-images.githubusercontent.com/9073066/45653606-8315ad80-bad0-11e8-827d-766e50c0f3da.png)
Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self' * 'unsafe-inline'".

    at Function (<anonymous>)
    at c029a6b2088014967102f8efaaf260dc7e0d3df5.js?meteor_js_resource=true:1
    at c029a6b2088014967102f8efaaf260dc7e0d3df5.js?meteor_js_resource=true:1
    at c029a6b2088014967102f8efaaf260dc7e0d3df5.js?meteor_js_resource=true:1
    at c029a6b2088014967102f8efaaf260dc7e0d3df5.js?meteor_js_resource=true:1
c029a6b2088014967102f8efaaf260dc7e0d3df5.js:1 Uncaught ReferenceError: RocketChat is not defined
    at c029a6b2088014967102f8efaaf260dc7e0d3df5.js:1
    at a (c029a6b2088014967102f8efaaf260dc7e0d3df5.js:1)
    at HTMLDocument.u (c029a6b2088014967102f8efaaf260dc7e0d3df5.js:1)
c029a6b2088014967102f8efaaf260dc7e0d3df5.js?meteor_js_resource=true:1 WebSocket connection to 'wss://chat.talkwithstranger.com/sockjs/127/ff4o9sea/websocket' failed: Error in connection establishment: net::ERR_NOT_IMPLEMENTED

**Comment by bernardobugmann** [comment]
Happened to me too when I upgrade to 0.69.2 release.

**Comment by fais3000** [comment]
Seems to be an issue with the Adblock extension.

**Comment by FaizanZahid** [comment]
Agree, seems to be only happending when user's adblock is installed and enabled. Can we have a fix for this please?

Adblock seems to be blocking some of the meteor scripts in chrome.

**Comment by obonyojimmy** [comment]
i can confirm same , Login page , registration page all blank on firefox 74.0
